packages/@react-spectrum/button/stories/ToggleButton.stories.tsx

```diff
@@ -84,3 +87,14 @@ function render(props = {}) {
     </ToggleButton>
   </Flex>);
 }
+
+function ControlledToggleButton() {
+  let [selected, setSelected] = useState(false);
+  return (
+    <div>
+      <ToggleButton onChange={setSelected}>Press Me</ToggleButton>
```

Comment:
Could we change this too?
```suggestion
      <ToggleButton isSelected={selected} onChange={setSelected}>Press Me</ToggleButton>
```